internal/cmdinit/cmdinit_test.go

```diff
@@ -51,24 +52,40 @@ packageMetadata:
 
 	b, err = ioutil.ReadFile(filepath.Join(d, "my-pkg", man.ManFilename))
 	assert.NoError(t, err)
-	assert.Equal(t, `my-pkg
-==================================================
+	assert.Equal(t, strings.ReplaceAll(`# my-pkg
 
-# NAME
+## Description
+my description
 
-  my-pkg
+## Usage
 
-# SYNOPSIS
+### Fetch the package
+'kpt pkg get REPO_URI[.git]/PKG_PATH[@VERSION] my-pkg'
+Details: https://googlecontainertools.github.io/kpt/reference/pkg/get/
 
-  kubectl apply --recursive -f my-pkg
+### Update package from upstream
+'kpt pkg update my-pkg[@VERSION] --strategy=resource-merge'
```

Comment:
if `resource-merge` is the typical strategy we're recommending to users, why isn't it the default?
As a general rule, make the 80% use case simpler

Reply:
Not sure what is the best default here, but worth investigating. Created issue https://github.com/GoogleContainerTools/kpt/issues/1033